blocks/hero-banner/hero-banner.css

```diff
@@ -100,6 +100,17 @@
     margin: 1rem auto;
 }
 
+.hero-banner.text-with-no-background-desktop p {
```

Comment:
We should avoid having `desktop` in class name I think. If these are only to be applied in `desktop` you can enforce that through media queries.

Reply:
removed -desktop in the class name